lms/static/js/spec/search/search_spec.js

```diff
@@ -348,7 +348,6 @@ define([
             this.listView.render();
             expect(this.listView.$el.find('ol')[0]).toExist();
             expect(this.listView.$el.find('li').length).toEqual(1);
-            expect(this.listView.$el).toContainHtml('Test Course');
```

Comment:
Why are you removing this line? You don't seem to be removing a "Test Course" string from the UI in this pull request.